assets/javascript/showMore.js

```diff
@@ -1,35 +1,37 @@
-const showMore = $('.show-more-tiles');
-const smallTiles = $('#small-tiles');
-function updateButton(isLastPage) {
-  if (isLastPage) {
-    showMore.remove();
-  }
-}
+const setUpShowMore = (showMoreButton, showMoreTiles, pageType = '') => {
+  const updateButton = isLastPage => {
+    if (isLastPage) {
+      showMoreButton.remove();
+    }
+  };
+  const enableShowMore = () => {
+    showMoreButton.html('Show more');
+    showMoreButton.attr('disabled', false);
+  };
+  const disableShowMore = () => {
+    showMoreButton.html('Loading');
+    showMoreButton.attr('disabled', true);
+  };
 
-function enableShowMore() {
-  showMore.html('Show more');
-  showMore.attr('disabled', false);
-}
+  let page = 1;
+  showMoreButton.on('click', function () {
+    page++;
+    const nextPage = `${location.pathname}.json?page=${page}&pageType=${pageType}`;
+    disableShowMore();
 
-function disableShowMore() {
-  showMore.html('Loading');
-  showMore.attr('disabled', true);
-}
-
-let page = 1;
-showMore.on('click', function () {
-  page++;
-  const nextPage = `${location.pathname}.json?page=${page}`;
-  disableShowMore();
-
-  $.getJSON(nextPage, function (data) {
-    const res = data.relatedContent.data.map(item =>
-      nunjucks.render('content-tile-small', { params: item }),
-    );
-    smallTiles.append(res);
-    enableShowMore();
-    updateButton(data.relatedContent.isLastPage);
-  }).fail(function () {
-    enableShowMore();
+    $.getJSON(nextPage, function (response) {
+      console.log('111111111');
+      console.log(response);
+      console.log('222222222');
+      const { data, isLastPage } = response;
+      const res = data.map(item =>
```

Comment:
already existed but a bit inconsistent with use of arrow functions and normal functions.. are we good just to use arrow functions?